lerna-package-lock.json

```diff
@@ -28809,7 +32518,12 @@
 		"is-lambda": {
 			"version": "1.0.1",
 			"resolved": "https://registry.npmjs.org/is-lambda/-/is-lambda-1.0.1.tgz",
+<<<<<<< HEAD
```

Comment:
Looks like you messed up resolving the conflicts. I recommend recreating this change from scratch (rever the lock file to how it is on main, delete your node modules folder, merge in main, npm i, make sure the lock file is unchanged, then update lerna again and install.

Reply:
Ah I've seen VSCode do this to me before. Let me fix it up.

Reply:
I'll re-do from scratch to make sure, but I just forgot to hit save after resolving conflicts

Reply:
Ya, it's gotten me with that before too. VS code has clarity issues around which things save and which don't.